packages/webdriverio/src/commands/element/addValue.ts

```diff
@@ -26,18 +38,22 @@ export type AddValueOptions = {
  * </example>
  *
  * @alias element.addValue
- * @param {string | number | boolean | object | Array<any>}      value     value to be added
- * @param {AddValueOptions=} options                    command options (optional)
- * @param {boolean}         options.translateToUnicode enable translation string to unicode value automatically
+ * @param {string | number | Array<string | number>}        value                       value to be added
+ * @param {CommandOptions=}                                 options                     command options (optional)
+ * @param {boolean}                                         options.translateToUnicode  enable translation string to unicode value automatically
  *
  */
 export default function addValue (
     this: WebdriverIO.Element,
-    value: string | number | boolean | object | Array<any>,
-    { translateToUnicode = true }: AddValueOptions = {}
+    value: Value | Value[],
+    { translateToUnicode = true }: CommandOptions = {}
 ) {
+    if (!isValidType(value)) {
+        new DeprecatedWarning('Support for values that are not of type "string", "number" or "Array<string | number>" will soon be dropped')
```

Comment:
Do you want to log or throw here? Right now I assume it doesn't do anything.

Reply:
Ah now I have seen the class for it. Is there any advantage of wrapping this within a class to create an instance that is immediately remove from GC? Wouldn't a simple:
```suggestion
        log.warn('@DeprecatedWarning: Support for values that are not of type "string", "number" or "Array<string | number>" will soon be dropped')
```
do the same?